crucible/src/Lang/Crucible/Analysis/Taint.hs

```diff
@@ -0,0 +1,158 @@
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TupleSections #-}
+
+module Lang.Crucible.Analysis.Taint (cfgTaintAnalysis
+                                    ,taintDomain
+                                    ,taintInterp
+                                    ,Tainted(..)) where
```

Comment:
I don't think this is actually used anywhere.  It is neat, but maybe we should just not include it since there are no clients?

Reply:
It's also unfinished, now that I look more closely. I'm inclined to leave the file there, so we can finish it at some point, but to remove it from the exported modules list (as in your comment below).